test/parallel/test-http-set-timeout-server.js

```diff
@@ -117,20 +117,25 @@ test(function serverRequestNotTimeoutAfterEnd(cb) {
 
 test(function serverResponseTimeoutWithPipeline(cb) {
   let caughtTimeout = '';
+  let secReceived = false;
   process.on('exit', function() {
     assert.strictEqual(caughtTimeout, '/2');
   });
   const server = http.createServer(function(req, res) {
```

Comment:
I thought of that but it's tricky because the number of times the callbacks are run depends on when the timer fires. I can wrap the `http.createServer()` callback though